packages/minifier-css/minifier-tests.js

```diff
@@ -92,9 +92,15 @@ Tinytest.add(
   (test) => {
     const css1 = '@import "custom.css"; body { color: "red"; }';
     const css2 = 'body { color: "blue"; }';
-    const cssAst1 = CssTools.parseCss(css1);
-    const cssAst2 = CssTools.parseCss(css2);
+    const cssAst1 = CssTools.parseCss(css1, {from: "test.css"});
+    const cssAst2 = CssTools.parseCss(css2, {from: "test2.css"});
     const mergedAst = CssTools.mergeCssAsts([cssAst1, cssAst2]);
+    const stringifiedAsts = CssTools.stringifyCss(mergedAst, {
```

Comment:
this is a test for the problems i faced with https://github.com/meteor/meteor/pull/10260/files#diff-bce46ba8fd6b413c9e323e42376b3374R59 applied - i didn't yet figure out how to solve it though.
The old `postcss()` variant works but spams errors - found a similar issue at https://github.com/stylelint/stylelint/issues/3477 but didn't fully understand how they solved it.